### PR TITLE
Add release notes for `v1.0.0-RC2`

### PR DESCRIPTION
**Pull request by lwronski**

### CODE REVIEW

File: website/docs/release_notes.md [hunk 1]
```diff
@@ -7,6 +7,189 @@ import ReactPlayer from 'react-player'
...
+## What's Changed
+
+### Exclude
+
+To exclude specific source files or entire directories from a Scala CLI project using the exclude directive in your `project.scala` file or the command line parameter `--exclude`, along with a pattern:
+- absolute path: `/root/path/to/your/project/Main.scala`
+- relative path: `src/main/scala/Main.scala`
+- glob pattern: `*.sc`
+
+For example, to exclude all files in the` example/scala` directory, add the following directive to your `project.scala` file:
```
Comment (bishabosha) [inline website/docs/release_notes.md]:
```suggestion
For example, to exclude all files in the `example/scala` directory, add the following directive to your `project.scala` file:
```

File: website/docs/release_notes.md [hunk 2]
```diff
@@ -7,6 +7,189 @@ import ReactPlayer from 'react-player'
...
+```scala
+//> using exclude "example/scala"
+```
+
+Added by [@lwronski](https://github.com/lwronski) in [#2053](https://github.com/VirtusLab/scala-cli/pull/2053).
+
+### Directives with a Test Scope Equivalent
```
Comment (Gedochao) [inline website/docs/release_notes.md]:
```suggestion
### Directives with a Test Scope equivalent
```

File: website/docs/release_notes.md [hunk 3]
```diff
@@ -7,6 +7,189 @@ import ReactPlayer from 'react-player'
...
+
+Some directives now have a test scope equivalent, such as `using dep` and its test scope counterpart `using test.dep`. This allows you to declare dependencies that are only used in tests outside of test-specific sources.
+
+For example, you can declare a dependency on munit in your `project.scala` file like this:
```
Comment (Gedochao) [inline website/docs/release_notes.md]:
```suggestion
For example, you can declare a dependency on `munit` in your `project.scala` file like this:
```

File: website/docs/release_notes.md [hunk 4]
```diff
@@ -7,6 +7,189 @@ import ReactPlayer from 'react-player'
...
+```
```
Comment (Gedochao) [inline website/docs/release_notes.md]:
````suggestion
```scala
````

File: website/docs/release_notes.md [hunk 5]
```diff
@@ -7,6 +7,189 @@ import ReactPlayer from 'react-player'
...
+//> using test.dep "org.scalameta::munit::0.7.29"
+```
+The dependency will only be available in test sources.
+
+Here's a list of directives with a test scope equivalent with example values:
+
+```
```
Comment (Gedochao) [inline website/docs/release_notes.md]:
````suggestion
```scala
````

File: website/docs/release_notes.md [hunk 6]
```diff
@@ -7,6 +7,189 @@ import ReactPlayer from 'react-player'
...
+ //> using test.toolkit "latest"
+```
+
+Added by [@Gedochao](https://github.com/Gedochao) in  [#2046](https://github.com/VirtusLab/scala-cli/pull/2046)
+
+### Changes to using-directives syntax
+
+We've made several updates to simplify the using directives syntax in this release:
+
+- removed the possibility of nesting from the API.
```
Comment (Gedochao) [inline website/docs/release_notes.md]: is this point relevant to Scala CLI users?

File: website/docs/release_notes.md [hunk 7]
```diff
@@ -7,6 +7,189 @@ import ReactPlayer from 'react-player'
...
+- allowed omitting commas in lists of values.
+- disallowed multiline comments.
+- removed multiline strings.
+- removed `require` and `@require` syntax support.
+- allowed values without quotes.
+- removed `@using`.
+
+
+Added by [@tgodzik](https://github.com/tgodzik) in [#2076](https://github.com/VirtusLab/scala-cli/pull/2076)
+
```
Comment (Gedochao) [inline website/docs/release_notes.md]:
```suggestion

Added by [@tgodzik](https://github.com/tgodzik) in [#2076](https://github.com/VirtusLab/scala-cli/pull/2076)
```

File: website/docs/release_notes.md [hunk 8]
```diff
@@ -7,6 +7,189 @@ import ReactPlayer from 'react-player'
...
+### Changes to using-directives syntax
+
+We've made several updates to simplify the using directives syntax in this release:
+
+- removed the possibility of nesting from the API.
+- allowed omitting commas in lists of values.
+- disallowed multiline comments.
+- removed multiline strings.
+- removed `require` and `@require` syntax support.
+- allowed values without quotes.
```
Comment (Gedochao) [inline website/docs/release_notes.md:62]: I think it'd be awesome to include an example showing that commas and quotes are now optional.
maybe something like:
````
```scala
//> using scala 3.2.2
//> using javacOpt source 1.8 target 1.8
```
````

File: website/docs/release_notes.md [hunk 9]
```diff
@@ -7,6 +7,189 @@ import ReactPlayer from 'react-player'
...
+Added by [@tgodzik](https://github.com/tgodzik) in [#2076](https://github.com/VirtusLab/scala-cli/pull/2076)
+
+
+### Bootstrapped standalone fat JAR.
+
+The Scala CLI launcher is available as a standalone fat JAR. You can download the nightly version of the Scala CLI fat JAR from Sonatype and try it now:
+
+```bash ignore
+cs launch org.virtuslab.scala-cli:cliBootstrapped:1.0.0-RC2 -M scala.cli.ScalaCli
+```
```
Comment (Gedochao) [inline website/docs/release_notes.md]: The description suggests using a nightly version, but the example points to `1.0.0-RC2`...?
or is it either the nightly version or `1.0.0-RC2`? 
in the case of the latter, we need 2 examples, I think

File: website/docs/release_notes.md [hunk 10]
```diff
@@ -7,6 +7,189 @@ import ReactPlayer from 'react-player'
...
+
+Added by [@lwronski](https://github.com/lwronski) in [#2040](https://github.com/VirtusLab/scala-cli/pull/2040)
+
+### Fix deadlocks in Script Wrappers
+
+We have resolved an issue that caused deadlocks when threads were run from the static initializer of the wrapper object 
+([#532](https://github.com/VirtusLab/scala-cli/pull/532) and [#1933](https://github.com/VirtusLab/scala-cli/pull/1933)).
+Based on the feedback from the community (Thanks [@dacr](https://github.com/dacr)), we found that encapsulating the script code
+into a class wrapper fixes the issue. The wrapper is generated by the Scala CLI and is not visible to the user.
```
Comment (Gedochao) [inline website/docs/release_notes.md:125]: It might be worth mentioning the behaviour change when `@main` is used, maybe with an output error example

File: website/docs/release_notes.md [hunk 11]
```diff
@@ -7,6 +7,189 @@ import ReactPlayer from 'react-player'
...
+## [v1.0.0-RC2](https://github.com/VirtusLab/scala-cli/releases/tag/v1.0.0-RC2)
+
+## What's Changed
+
+### Exclude
+
+To exclude specific source files or entire directories from a Scala CLI project using the exclude directive in your `project.scala` file or the command line parameter `--exclude`, along with a pattern:
+- absolute path: `/root/path/to/your/project/Main.scala`
+- relative path: `src/main/scala/Main.scala`
+- glob pattern: `*.sc`
```
Comment (Gedochao) [inline website/docs/release_notes.md]:
```suggestion
To exclude specific source files or entire directories from a Scala CLI project, you can now us the `//> using exclude` directive in your `project.scala` file. Alternatively, you can do the same from the command line with the `--exclude` option.
Accepted pattern syntax:
- absolute path: `/root/path/to/your/project/Main.scala`
- relative path: `src/main/scala/Main.scala`
- glob pattern: `*.sc`
```

File: website/docs/release_notes.md [hunk 12]
```diff
@@ -7,6 +7,189 @@ import ReactPlayer from 'react-player'
...
+```bash ignore
+cs launch org.virtuslab.scala-cli:cliBootstrapped:1.0.0-RC2 -M scala.cli.ScalaCli
+```
+
+Added by [@romanowski](https://github.com/romanowski) in [#2005](https://github.com/VirtusLab/scala-cli/pull/2005).
+
+
+### Access the Name of the Running Script
+
+With the special `scriptPath` function, you can now easily access the name of the running script inside the script itself. Example of how to use the scriptPath variable:
```
Comment (Gedochao) [inline website/docs/release_notes.md]:
```suggestion
With the special `scriptPath` function, you can now easily access the name of the running script inside the script itself. Example of how to use the `scriptPath` value:
```

File: website/docs/release_notes.md [hunk 13]
```diff
@@ -7,6 +7,189 @@ import ReactPlayer from 'react-player'
...
+
+</ChainedSnippets>
+
+Added by [@lwronski](https://github.com/lwronski) in [#1990](https://github.com/VirtusLab/scala-cli/pull/1990)
+
+### Explicit Handling of Paths in using-directives
+
+The `${.}` pattern in directive values can now be replaced by the parent directory of the file containing the directive. This makes it possible to generate coverage output files relative to the source file location, for example:
+
+```
```
Comment (Gedochao) [inline website/docs/release_notes.md]:
````suggestion
```scala
````

File: website/docs/release_notes.md (hunk 6 above)
Comment (lwronski) [reply website/docs/release_notes.md]: I don't think, so I remove it.

File: website/docs/release_notes.md [hunk 14]
```diff
@@ -7,6 +7,189 @@ import ReactPlayer from 'react-player'
...
+The Scala CLI launcher is available as a standalone fat JAR. You can download the nightly version of the Scala CLI fat JAR from Sonatype and try it now:
+
+```bash ignore
+cs launch org.virtuslab.scala-cli:cliBootstrapped:1.0.0-RC2 -M scala.cli.ScalaCli
+```
+
+Added by [@romanowski](https://github.com/romanowski) in [#2005](https://github.com/VirtusLab/scala-cli/pull/2005).
+
+
+### Access the Name of the Running Script
```
Comment (Gedochao) [inline website/docs/release_notes.md]:
```suggestion
### Access the path of the script being run from its code
```

File: website/docs/release_notes.md (hunk 12 above)
Comment (Gedochao) [inline website/docs/release_notes.md]:
```suggestion
With the special `scriptPath` function, you can now easily access the path of the script being run from inside the script itself. Example of how to use the `scriptPath` value:
```

File: website/docs/release_notes.md (hunk 9 above)
Comment (lwronski) [reply website/docs/release_notes.md]: I think we shouldn't mention about nightly version.  I have updated this description and it now points to the stable version.

File: website/docs/release_notes.md [hunk 15]
```diff
@@ -7,6 +7,218 @@ import ReactPlayer from 'react-player'
...
+
+```bash ignore
+cs launch org.virtuslab.scala-cli:cliBootstrapped:1.0.0-RC2 -M scala.cli.ScalaCli
+```
+
+Added by [@romanowski](https://github.com/romanowski) in [#2005](https://github.com/VirtusLab/scala-cli/pull/2005).
+
+### Access the path of the script being run from its code
+
+With the special `scriptPath` function, you can now easily access the path of the being run from script inside the script itself. Example of how to use the `scriptPath` variable:
```
Comment (Gedochao) [inline website/docs/release_notes.md]:
```suggestion
With the special `scriptPath` function, you can now easily access the path of the script being run from the script code itself. 
Here's an example of how to use the `scriptPath` value:
```

File: website/docs/release_notes.md [hunk 16]
```diff
@@ -7,6 +7,218 @@ import ReactPlayer from 'react-player'
...
+- removed multiline strings.
+- removed `require` and `@require` syntax support.
+- allowed values without quotes.
+- removed `@using`.
+
+For example, the following using directives are now valid without the need for commas and quotes:
+
+```scala
+//> using scala 3.2.2
+//> using javacOpt source 1.8 target 1.8
```
Comment (tgodzik) [inline website/docs/release_notes.md]:
```suggestion
//> using javacOpt -source 1.8 -target 1.8
```

**Review by tgodzik — APPROVED** [review]
